src/reducers/onboarding-status.reducer.js

```diff
@@ -23,9 +24,9 @@ export type State =
 const initialState = 'brand-new';
 
 export default (state: State = initialState, action: Action) => {
-  if (state === 'done') {
-    return state;
-  }
+  // if (state === 'done') { // Reset all not working with this --> does this break something?
+  //   return state;
+  // }
```

Comment:
Ahhh, that's the issue. Sorry about that, this was short-sighted of me. Was a premature optimization, since I assumed once onboarding was done, we'd never need to update this reducer again :(

We can remove it without issue